src/UI/Components/TreeTable/TreeRow/InlineInput.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import { Button, Icon, Switch, TextInput } from "@patternfly/react-core";
+import { CheckIcon } from "@patternfly/react-icons";
+import styled from "styled-components";
+import { words } from "@/UI/words";
+const InlineInput = ({
+  label,
+  value,
+  type,
+  onChange,
+  toggleModal,
+}: {
+  label: string;
+  value: string | number | boolean;
+  type: string;
+  onChange: (value: string | number | boolean) => void;
+  toggleModal: () => void;
+}) => {
+  return (
+    <>
+      {type.toLowerCase().includes("bool") ? (
+        <StyledSwitch
+          isChecked={value.toString().toLowerCase() === "true"}
+          onChange={(checked) => onChange(checked)}
+          aria-label={`new-attribute-toggle-${label}`}
+        />
+      ) : (
+        <StyledInput
+          value={value.toString()}
+          type={type.toLowerCase().includes("int") ? "number" : "text"}
+          onChange={(value) => onChange(value)}
+          aria-label="new-attribute-input"
+          placeholder={words("inventory.editAttribute.placeholder")}
+        />
+      )}
+
```

Comment:
why is there a new line here ?

Reply:
I extracted that code from CellWithCopyExpert and I didn't notice extra space